Makefile

```diff
@@ -99,3 +100,7 @@ tidy-all:
 .PHONY: fmt-all
 fmt-all:
 	$(MAKE) -j 4 -C $(PKG_RECEIVER_DIRS) fmt
+
+.PHONY: checks
+checks:
+	$(MAKE) -j 4 $(CHECKS)
```

Comment:
I'm thinking we should have a message after here saying "commit the result cuz things changed" like the git diff stuff we run